### DISCUSSION TITLE
Installing Debian on WS5220

### DISCUSSION THREAD

**Discussion by robertoand** [1000001101000/Debian_on_Intel_Terastations#10] [General]
HI,

I have a WS5220 8TB TeraStation on which I would like to install Debian instead of Windows Server 2006. I have seen the fantastic job you have done to integrate the operating system with the various hardware features of the Intel based Terastations. I read that my model is tested, but honestly I can't figure out how to install it. The WS5220 doesn't have a boot selector switch to assign priority to USB devices, there is the little red reset button under the display (or I think it's reset because the manual doesn't mention it at all) and the recovery procedure from the supplied SD card (with Windows Server 2016 and various Buffalo programs pre-installed, just pressing the "function" key it does it by itself). Also, I have no experience with SSH commands and wouldn't even know where to start (btw all my computers are Macs), so I'd install it the traditional way (connected to a VGA monitor and booting from your installer). Problem is, I don't know how to invoke the boot from the ISO you posted, but I assume I'll have to plug a keyboard into it and hit the usual keys that lead to BIOS or BOOT device selection. I would also try via SSH but I don't have any experience with that. When you have time, it would be useful to update the guide a little more specifically to make the step-by-step installation within everyone's reach. Thank you for the effort you put into this project!

Greetings from Italy!
Roberto

**Comment by 1000001101000** [comment]
If you wipe the drives before you start it should boot from the usb device (since that would be the only boot device it finds). Otherwise connecting a keyboard and monitor should give you access to the bios boot menu as you suspect.

Once you boot the installer which you can confirm via the front lcd or attached monitor it just becomes a matter of connecting with your favorite ssh client and following the menus.

**Reply by robertoand** [reply]
Thanks for the very fast reply. Once I've inserted the wiped disks (I'll use the Mac's diskutil erasedisk command) and the system boots from your ISO, what should I do? Do I install an SSH Client on my mac and then recall the IP of the Terastation (I've already assigned it a static IP from my router) and continue with the commands? Sorry, but I've NEVER used SSH and I don't know how it works.

**Comment by 1000001101000** [comment]
That sounds about right.

**Reply by robertoand** [reply]
Okay, thank you very much. I will try in the next few days!
